packages/peregrine/lib/targets/peregrine-declare.js

```diff
@@ -18,33 +47,35 @@ module.exports = targets => {
          * a talon.
          *
          *
-         * @member {tapable.SyncHook}
+         * @member {tapable.AsyncSeriesHook}
          *
          * @see [list of wrappable talons][]
          *
-         * @see [Intercept function signature]{@link talonInterceptFunction}
+         * @see [Intercept function signature]{@link hookInterceptFunction}
          *
          * @example <caption>Access the tapable object</caption>
          * const peregrineTargets = targets.of('@magento/peregrine');
          * const talonsTarget = peregrineTargets.talons;
          *
          * @example <caption>Wrap the `useApp()` hook  with a logging extension</caption>
          *
-         * talonsTarget.tap(talonWrapperConfig => {
-         *   talonWrapperConfig.App.useApp.wrapWith('@my-extensions/log-wrapper');
+         * talonsTarget.tap(talons => {
+         *   talons.App.useApp.wrapWith('@my-extensions/log-wrapper');
          * })
          */
-        talons: new targets.types.Sync(['talons'])
+        talons: new targets.types.AsyncSeries(['talons'])
```

Comment:
Note that changing a target from Sync to Async is a backwards-compatible change for interceptors. They can still use `tap()` with a synchronous interceptor.